hackhub/data-mesh-hack/scripts/infra/bicep/azure-sql.bicep

```diff
@@ -39,9 +39,16 @@ resource sqlserver_CloudSales 'Microsoft.Sql/servers/databases@2022-11-01-previe
   tags: {
     displayName: cloudSalesDbName
   }
+  sku: {
+    name: 'GP_S_Gen5_2'
+    tier: 'GeneralPurpose'
+  }
```

Comment:
Do we need to add more here - something like `capacity` to ensure that it chooses specified SKU in serverless compute mode as opposed to provisioned mode?

Reply:
No, everything is included here.